src/host/ft_host/API_MultipleInflightMessageTests.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "precomp.h"
+
+class MultipleInflightMessageTests

```

Comment:
It might be nice to add a comment here, explaining that this test ensures that no regressions occurs, since this bug was fixed in `c0ab9cb5b55d745a1528c7429615c71e0745ed8d`.

Reply:
done! thanks!